### PR TITLE
Yessenia Rodriguez css-positioning

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-01-22 at 10 46 04 AM
Screen Shot 2021-01-22 at 11 12 57 AM
Screen Shot 2021-01-22 at 11 17 56 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+
+  /* your code here */
+  position: absolute;
+  left: 230px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:21]: Please revise this to use smaller measurements from `right` as we discussed in Q&A

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Very nice so far!! One minor note:

File: css-positioning/absolute.css (hunk 1 above)
Comment (yesseniarod) [reply css-positioning/absolute.css]: Screen Shot 2021-01-22 at 3 16 33 PM

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**